tests/unit/test-class-wc-payment-gateway-wcpay.php

```diff
@@ -426,6 +426,101 @@ public function test_process_refund_with_reason_non_usd() {
 		$this->assertTrue( $result );
 	}
 
+	public function test_process_refund_interac_present() {
```

Comment:
This test fails due to [the addition of the refund ID to the notes](https://github.com/Automattic/woocommerce-payments/blob/0af1f4305851f30aa38a0f598d79d6442288f8be/includes/class-wc-payment-gateway-wcpay.php#L1520), I'm looking at how to resolve that for `interac_present` transactions, and whether we should fetch a refund from Stripe on the server to store somewhere too.